README.md

```diff
@@ -1,5 +1,8 @@
 # SHAP Community Fork
 
+![example workflow](https://github.com/dsgibbons/shap/actions/workflows/run_tests.yml/badge.svg)
```

Comment:
Suggest specifying the master branch for this badge

Reply:
You mean like this? 

`https://github.com/dsgibbons/shap/actions/workflows/run_tests.yml/badge.svg?branch=master`

Just curious, if we don't specify a branch, what branch does it refer to for the badge?

Reply:
Yes that was what I was thinking
 I'm unsure exactly what happens otherwise, I think it would be the most recent workflow run which might be from a PR.